### DISCUSSION TITLE
Popup doesn't go away when I select metamask

### DISCUSSION THREAD

**Discussion by GrahamMcBain** [WalletConnect/walletconnect-monorepo#1568] [Developer Support]
I have implemented web3modal on our site and clicking the connect button brings up the popup to select which wallet to connect with but once click metamask and login the pop up window does not go away. When you refresh the page once the popup is stuck there then the site is connected. Not sure why the pop up isn't going away after login...

**Comment by xzilja** [comment]
Can you replicate this issue on https://beta-2.web3modal-dev.pages.dev ? Also any chance you can provide repo with reproduction, atm hard to tell what could be causing this
